code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -86,9 +89,29 @@
 /datum/reagent/proc/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
 	return
 
-/datum/reagent/proc/overdose(var/mob/living/carbon/M, var/alien) // Overdose effect. Doesn't happen instantly.
-	M.adjustToxLoss(REM)
-	return
+/datum/reagent/proc/overdose(var/mob/living/carbon/M, var/alien)
+//	M.overdosing -= REM
+	if(od_type == "heart")
+		if (istype(M, /mob/living/carbon/human))
+			var/mob/living/carbon/human/C = M
+			var/obj/item/organ/internal/heart/H = C.internal_organs_by_name[O_HEART]
+			playsound(M.loc, 'sound/effects/singlebeat.ogg', 5, -1)	//Don't know how to keep this mob specific
+			C.pulse += 2
+			if (prob (5))
+				H.damage += 1
+				M << "<span class='danger'>You feel a stabbing pain in your chest!</span>"
+		else
+			M.adjustToxLoss(2)
+		return
+	else
+		if (istype(M, /mob/living/carbon/human))
+			var/mob/living/carbon/human/C = M
+			var/obj/item/organ/internal/liver/L = C.internal_organs_by_name[O_LIVER]
+			L.damage += rand(3,6)

```

Comment:
Constant number kills way too hard with heart damage, plus this makes it feel like the heart is tearing itself apart, instead of slowly rotting away.
As for the liver bit, I couldn't figure out a good number that'd make OD scary, but not too scary, and the 3-6 felt right when I was testing it.